### PR TITLE
Port pocket multitool from gt6

### PR DESCRIPTION
**Pull request by iouter**
Pocket Multitool small

code and texture from gt6.

https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6640

### CODE REVIEW

**Review by leagris — CHANGES_REQUESTED** [review]
You need to revert those improper format changes on the gradle files. Otherwise it is all good.

File: addon.gradle [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 test {
-    useJUnitPlatform()
-    testLogging {
-        events "passed", "skipped", "failed"
-    }
+	useJUnitPlatform()
+	testLogging {
+		events "passed", "skipped", "failed"
+	}
 }
```
[the remark this answers is not in the document]
Comment (iouter) [reply addon.gradle]: done

**Review by boubou19 — APPROVED** [review]

**Comment by GTNHAfx237v7** [comment]
How does this tool work in terms of durability? Or is it unbreakable? I assume this is a non-electric version?

**Comment by iouter** [comment]
> How does this tool work in terms of durability? Or is it unbreakable? I assume this is a non-electric version?

same as other gt tools

**Review by chochem — CHANGES_REQUESTED** [review]
I am a bit confused that this is already approved. Can you please add some more information?

What tier is it supposed to be available? whats the recipe? which tools does it cover? If its a saw, does it work in the TGS? Can it go into the toolbox? Is it electric or does it have durability, your previous reply did not actual answer that?

**Comment by iouter** [comment]
> I am a bit confused that this is already approved. Can you please add some more information?
> 
> What tier is it supposed to be available? whats the recipe? which tools does it cover? If its a saw, does it work in the TGS? Can it go into the toolbox? Is it electric or does it have durability, your previous reply did not actual answer that?

1. a long rod, a file head, a saw blade, a plate, 2 rings, 2 sword blades and a blue dye. 
image
2. knife, saw, file, screwdriver, wire cutter, branch cutter.
3. no
4. yes
5. durability

**Review by chochem — APPROVED** [review]
thanks! That sounds fine.

**Comment by iouter** [comment]
sry, i forgot a question, because the file head can only be made with a mv extruder now, it needs mv, other parts can be made in the stone age.

**Comment by leagris** [comment]
And thank you @iouter for this nice GT6 port contribution. Really appreciate all the attention and time you spent for it.

**Pull request merged**